src/_gettsim/transfers/arbeitsl_geld_2/kindergelduebertrag.py

```diff
@@ -0,0 +1,113 @@
+"""Module for the calculation of the Kindergeldübertrag."""
+
+import numpy as np
+
+from _gettsim.shared import join_numpy, policy_info
+
+aggregate_by_p_id_kindergeldübertrag = {
+    "kindergeld_übertrag_m": {
+        "p_id_to_aggregate_by": "p_id_kindergeld_empf",
+        "source_col": "_kindergeld_kindbedarf_differenz_m",
```

Comment:
```suggestion
        "source_col": "_diff_kindergeld_kindbedarf_m",
```
Easier to read, I'd say?